# Replaces the federated_language_jax_pypi Python dependency with the
# federated_language Bazel dependency.

diff --git federated_language_jax/backend/BUILD federated_language_jax/backend/BUILD
index b008774..1e55995 100644
--- federated_language_jax/backend/BUILD
+++ federated_language_jax/backend/BUILD
@@ -38,7 +38,7 @@ py_library(
     deps = [
         "//federated_language_jax/computation:jax_computation",
         "//federated_language_jax/executor:executor_factory",
-        "@federated_language_jax_pypi//federated_language",
+        "@federated_language//federated_language",
     ],
 )

@@ -48,8 +48,8 @@ py_test(
     deps = [
         ":execution_contexts",
         "//federated_language_jax/computation:jax_computation",
+        "@federated_language//federated_language",
         "@federated_language_jax_pypi//absl_py",
-        "@federated_language_jax_pypi//federated_language",
         "@federated_language_jax_pypi//numpy",
     ],
 )
diff --git federated_language_jax/computation/BUILD federated_language_jax/computation/BUILD
index 6d4f725..f411d04 100644
--- federated_language_jax/computation/BUILD
+++ federated_language_jax/computation/BUILD
@@ -35,8 +35,8 @@ py_library(
     visibility = ["//visibility:public"],
     deps = [
         ":jax_serialization",
+        "@federated_language//federated_language",
         "@federated_language_jax_pypi//dm_tree",
-        "@federated_language_jax_pypi//federated_language",
         "@federated_language_jax_pypi//jax",
         "@federated_language_jax_pypi//numpy",
     ],
@@ -47,8 +47,8 @@ py_test(
     srcs = ["jax_computation_test.py"],
     deps = [
         ":jax_computation",
+        "@federated_language//federated_language",
         "@federated_language_jax_pypi//absl_py",
-        "@federated_language_jax_pypi//federated_language",
         "@federated_language_jax_pypi//jax",
         "@federated_language_jax_pypi//ml_dtypes",
         "@federated_language_jax_pypi//numpy",
@@ -58,7 +58,7 @@ py_test(
 py_library(
     name = "jax_computation_context",
     srcs = ["jax_computation_context.py"],
-    deps = ["@federated_language_jax_pypi//federated_language"],
+    deps = ["@federated_language//federated_language"],
 )

 py_test(
@@ -78,9 +78,9 @@ py_library(
     deps = [
         ":jax_computation_context",
         ":xla_serialization",
+        "@federated_language//federated_language",
         "@federated_language//federated_language/common_libs:structure",
         "@federated_language//federated_language/proto:computation_py_pb2",
-        "@federated_language_jax_pypi//federated_language",
         "@federated_language_jax_pypi//jax",
         "@federated_language_jax_pypi//numpy",
     ],
@@ -91,10 +91,10 @@ py_test(
     srcs = ["jax_serialization_test.py"],
     deps = [
         ":jax_serialization",
+        "@federated_language//federated_language",
         "@federated_language//federated_language/common_libs:structure",
         "@federated_language//federated_language/proto:computation_py_pb2",
         "@federated_language_jax_pypi//absl_py",
-        "@federated_language_jax_pypi//federated_language",
         "@federated_language_jax_pypi//jax",
         "@federated_language_jax_pypi//numpy",
     ],
@@ -104,8 +104,8 @@ py_library(
     name = "xla_serialization",
     srcs = ["xla_serialization.py"],
     deps = [
+        "@federated_language//federated_language",
         "@federated_language//federated_language/proto:computation_py_pb2",
-        "@federated_language_jax_pypi//federated_language",
         "@federated_language_jax_pypi//protobuf",
         "@xla//xla/python:xla_client",
     ],
@@ -116,9 +116,9 @@ py_test(
     srcs = ["xla_serialization_test.py"],
     deps = [
         ":xla_serialization",
+        "@federated_language//federated_language",
         "@federated_language//federated_language/proto:computation_py_pb2",
         "@federated_language_jax_pypi//absl_py",
-        "@federated_language_jax_pypi//federated_language",
         "@federated_language_jax_pypi//jax",
         "@federated_language_jax_pypi//numpy",
         "@federated_language_jax_pypi//protobuf",
diff --git federated_language_jax/executor/BUILD federated_language_jax/executor/BUILD
index 997cbaf..c11fca0 100644
--- federated_language_jax/executor/BUILD
+++ federated_language_jax/executor/BUILD
@@ -39,9 +39,9 @@ py_library(
     srcs = ["executor_factory.py"],
     data = ["@org_tensorflow_federated//tensorflow_federated/cc/core/impl/executors:executor_bindings"],
     deps = [
+        "@federated_language//federated_language",
         "@federated_language_jax_pypi//absl_py",
         "@federated_language_jax_pypi//cachetools",
-        "@federated_language_jax_pypi//federated_language",
         "@org_tensorflow_federated//tensorflow_federated/python/core/impl/executors:cpp_to_python_executor",
     ],
 )
